src/components/MySongsTable.js

```diff
@@ -77,6 +52,11 @@ const styles = theme => ({
 });
 
 class MySongsTable extends React.Component {
```

Comment:
You can import { Component } at the top and write 
"class MySongsTable extends Component {"
So it will be aonsistant with the other pages in our application